packages/broker/src/plugins/operator/OperatorValueBreachWatcher.ts

```diff
@@ -0,0 +1,63 @@
+import { EthereumAddress, Logger, scheduleAtInterval } from '@streamr/utils'
+import { MaintainOperatorValueHelper } from './MaintainOperatorValueHelper'
+import { OperatorServiceConfig } from './OperatorPlugin'
+
+const logger = new Logger(module)
+
+const CHECK_VALUE_INTERVAL = 1000 * 60 * 60 // 1 hour
+
+export class OperatorValueBreachWatcher {
+    private penaltyLimitFractionCached?: bigint
+    private readonly abortController: AbortController
+    
+    // public access modifier for tests 
+    readonly helper: MaintainOperatorValueHelper
```

Comment:
It's not possible to deploy a new dev env for these tests (using streamrEnvDeployer) because we also need to query the existing subgraph. The alternative would be to build a new subgraph image on the fly for the newly generated addresses